tests/test_freshdesk_discovery.py

```diff
@@ -0,0 +1,122 @@
+import re
+
+from tap_tester import menagerie, connections
+
+from base import FreshdeskBaseTest
+
+
+class TestFreshdeskDiscovery(FreshdeskBaseTest):
+
+    def name(self):
+        return "tap_tester_freshdesk_discovery"
+
+    def test_run(self):
+        """
```

Comment:
Can we test below scenarios?
- Verify there are no duplicate/conflicting metadata entries.
- Verify replication key(s) match expectations.
- Verify the actual replication matches our expected replication method.
- Verify that if there is a replication key we are doing INCREMENTAL otherwise FULL.
- Verify all streams have inclusion of automatic

Reply:
All the assertions mentioned above are already added to this file.